subworkflows/local/fastqc_trimgalore.nf

```diff
@@ -0,0 +1,76 @@
+//
+// Read QC, and trimming
+//
+
+include { FASTQC           } from '../../modules/nf-core/fastqc/main'
```

Comment:
This can be installed via nf-core subworkflows

Reply:
Fixed in #77